src/coreclr/vm/stubmgr.cpp

```diff
@@ -1783,41 +1783,43 @@ BOOL ILStubManager::TraceManager(Thread *thread,
             return FALSE;
         }
 
-        LOG((LF_CORDB, LL_INFO1000, "ILSM::TraceManager: Step through to target - 0x%p\n", pTargetMD));
+        LOG((LF_CORDB, LL_INFO1000, "ILSM::TraceManager: Step through to target - %p\n", pTargetMD));
         target = GetStubTarget(pTargetMD);
         if (target == (PCODE)NULL)
             return FALSE;
 
         trace->InitForManaged(target);
     }
+    else if (pStubMD->IsPInvokeStub())
```

Comment:
Can we ever end up with pStubMD->IsPInvokeStub() == true here now that we are patching the MethodDesc in the code header?